layer.yaml

```diff
@@ -1,31 +1,59 @@
-includes: ['layer:hadoop-client', 'interface:spark', 'interface:benchmark']
+includes:
+- layer:hadoop-client
+- interface:spark
+- interface:benchmark
 options:
-  hadoop-client:
+  apache-spark:
```

Comment:
Hi Cory

Thanks for your thorough feedback. Can you explain a bit more what the idea
behind the options is? I thought this was just an upstreamed replacement
for dist.yaml. I'm not sure why these options schould be passed to the
hadoop-client. I am in fact setting them for and using them in the spark
layer. I'm not sure why these should be passed to another layer.

Op maandag 29 februari 2016 heeft Cory Johns <notifications@github.com> het
volgende geschreven:
> In layer.yaml:
>
>> options:
>> - hadoop-client:
>> + apache-spark:
>
> This isn't how layer options work. The layer name here is the layer for
which you are setting the options, or, in other words, the layer that
defines the options you want to set. In this case, that's hadoop-client and
setting options for any other layer (apache-spark, in this case) will cause
them to be ignored (it should probably complain during build instead).
>
> —
> Reply to this email directly or view it on GitHub.<
https://ci5.googleusercontent.com/proxy/C4LZzv7PlLgC2rWZmE6Z4GHtk3I0aJtZCCFZMzFVz--8DYfxjaezvrp7hT5fyqrIL3mP3FFp3hYDv_6KWdNteyXOQIz8lx2aT93guJk5NDKbd4zam7Si1Nv5V4FyNXk_wamVDnhWc1nLCdJC-8w3KjbSfEpKDQ=s0-d-e1-ft#https://github.com/notifications/beacon/ABbH9XVzH5ToQCxj6VcXX4qHzQimE-tGks5poz44gaJpZM4Hkmc3.gif
>